src/storage/mutate/AddVerticesProcessor.cpp

```diff
@@ -16,8 +16,9 @@ namespace nebula {
 namespace storage {
 
 void AddVerticesProcessor::process(const cpp2::AddVerticesRequest& req) {
-    auto version =
-        std::numeric_limits<int64_t>::max() - time::WallClock::fastNowInMicroSec();
+    auto version = FLAGS_enable_multi_versions ?
+        std::numeric_limits<int64_t>::max() - time::WallClock::fastNowInMicroSec() :
+        std::numeric_limits<int64_t>::max();
```

Comment:
it should be zero if FLAGS_enable_multi_versions is false

Reply:
Yeah, may bad, hold on, there are some other ut problems.